tests/specs/examples/vat.sol

```diff
@@ -0,0 +1,246 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+/// vat.sol -- Dai CDP database
+
+// Copyright (C) 2018 Rain <rainbreak@riseup.net>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.5.12;
+
+// FIXME: This contract was altered compared to the production version.
+// It doesn't use LibNote anymore.
+// New deployments of this contract will need to include custom events (TO DO).
+
+contract Vat {
```

Comment:
I think we better pick a different example to include in the repo here, AGPL is not license compatible with BSD 3-clause, which this repo basically is.

Reply:
Any contract here is a good choice: https://github.com/OpenZeppelin/openzeppelin-contracts

For example the tokens here: https://github.com/OpenZeppelin/openzeppelin-contracts/tree/7f6a1666fac8ecff5dd467d0938069bc221ea9e0/contracts/token

Any of these can be collapsed to a single file, and as long as the MIT license identifier remains at the top we remain license compatible.

Reply:
You can also just google for any MIT-ish licensed ERC-777 or ERC-721 (if you want something more complicated than an ERC-20). ERC-20 is also probably a fine choice.

Reply:
Added example for ERC-721.